src/Core/Services/StateMigrationService.cs

```diff
@@ -612,5 +905,17 @@ private IStorageService GetStorageService(Storage storage)
                     return _liteDbStorageService;
             }
         }
+
+        private async Task MoveToPrefsAsync<T>(string key)
+        {
+            var value = await GetValueAsync<T>(Storage.LiteDb, key);
+            if (value == null)
+            {
+                return;
+            }
+
+            await SetValueAsync(Storage.Prefs, key, value);
+            await RemoveValueAsync(Storage.LiteDb, key);
+        }
```

Comment:
🤔 Could it happen that the migration at some point fails for some reason and the user starts interacting with the app, which means that the new key/value are going to be stored in `Prefs` and then when the app starts again the migration runs smoothly overriding the last `Prefs` values that the user added with the ones with the old `LiteDB`values? 
E.g. 
v1 of the app has Solarized theme.
v2 of the app has the migration but the theme fails to be migrated for some exception. The user thought changes the theme to Nord.
v3 of the app fixes the migration and when run the theme will be Solarized again which would be wrong I think.

So what I'm getting at is, shall we check first to get the value for the key in `Prefs` and only if that value is `null` set the new value?
```suggestion
        private async Task MoveToPrefsAsync<T>(string key)
        {
            var value = await GetValueAsync<T>(Storage.LiteDb, key);
            if (value == null)
            {
                return;
            }

            if (await GetValueAsync<T>(Storage.Prefs, key) == null)
            {
                await SetValueAsync(Storage.Prefs, key, value);
            }
            await RemoveValueAsync(Storage.LiteDb, key);
        }
```